### ISSUE TITLE
Clear explanation of the textsum code maybe better

### ISSUE THREAD

**Issue by rylanchiu** [tensorflow/models#1729]
This issue concerns about the `batch_reader` in the `textsum` model. There are two for loops in line [71](https://github.com/tensorflow/models/blob/master/textsum/batch_reader.py#L71) and line [76](https://github.com/tensorflow/models/blob/master/textsum/batch_reader.py#L76) respectively. I am quite confused about these two for-loops. Maybe they are for reading more data given the toy data set? Maybe it is better to explain it in the code. Since maybe there is someone like me use this code to train the model on a large scale dataset. Then these two lines of code can be removed. Please correct me if I misunderstood the intention.

**Comment by rylanchiu** [comment]
Sorry, after reading the code again, I thought it may just create a number threads for multi-thread processing of batches. I made the guess about reusing the data since I found a tricky phenomenon during the training. After every n steps, the loss increased quite a lot and decreased again. And I have no idea why this happens. I would close this issue.
